app/Http/Controllers/Backend/IcsController.php

```diff
@@ -26,8 +26,9 @@ public static function generateIcsCalendar(
     ): Calendar {
         $icsToken = IcsToken::where([['token', $token], ['user_id', $user->id]])->firstOrFail();
 
-        $trainCheckIns = TrainCheckin::with(['HafasTrip.stopoversNEW'])
-                                     ->where('user_id', $user->id)
+        $trainCheckIns = TrainCheckin::where('user_id', $user->id)
+            //::with(['HafasTrip.stopoversNEW'])
+            //I don't know why, but the "with" eager loading doesn't work in prod. "HafasTrip" is always null then
```

Comment:
I don't want to merge this, but I don't have another plan... What's the problem with the eager loading? :(